rfc-035-use-standard-linting-tools-across-all-our-projects.md

```diff
@@ -0,0 +1,39 @@
+# Use standard linting tools across all of our projects
+
+## Summary
+
+Where available, all new dxw-owned or maintained repositories SHOULD use a
+linter for the programming language the repo is written in, where one is
+available.
+
+## Problem
+
+As developers, we commonly have different styles of writing code. Unchecked this
+can lead to inconsistencies in style, which over time can make code messy, and
+difficult to maintain and review. It can also lead to bikeshedding over code
+style in code reviews, which can cause conflict and slow down code review.
+
+## Proposal
+
+Where available, all new dxw-owned or maintained repositories SHOULD use a
+standard linting tool to make sure code meets a standard style. Examples of this
+include (but are not limited to):
```

Comment:
I suggest we further restrict to "an opinionated code linter and formatter". One of the motivations of this is to reduce the number of conversations about what the rules should be, and opinionated tools help by making those decisions for us. This is the main reason why I've pushed for Prettier for JS. If you can get over conforming to someone else's style, you stop having to think or talk about style ever again.